lib/bundler/source.rb

```diff
@@ -20,13 +20,21 @@ def version_message(spec)
         locked_spec = Bundler.locked_gems.specs.find {|s| s.name == spec.name }
         locked_spec_version = locked_spec.version if locked_spec
         if locked_spec_version && spec.version != locked_spec_version
-          message += Bundler.ui.add_color(" (was #{locked_spec_version})", :green)
+          message += Bundler.ui.add_color(" (was #{locked_spec_version})", earlier_version?(spec.version, locked_spec_version) ? :yellow : :green)
```

Comment:
```
earlier_version?(spec.version, locked_spec_version) ? :yellow : :green
```

We can probably extract this logic into a private method that returns the `:color`